src/devtools/client/webconsole/utils/autocomplete-eager.ts

```diff
@@ -49,3 +55,24 @@ export async function eagerEvaluateExpression(expression: string): Promise<Value
 
   return null;
 }
+
+export function useGetEvaluatedProperties() {
+  const frame = useSelector(getSelectedFrame);
+
+  if (!frame) {
+    return () => null;
+  }
+
+  return async (expression: string) =>
+    await getEvaluatedProperties(expression, frame.asyncIndex, frame.protocolId);
+}
+export function useEagerEvaluateExpression() {
+  const frame = useSelector(getSelectedFrame);
+
+  if (!frame) {
+    return () => null;
+  }
+
+  return async (expression: string) =>
+    await eagerEvaluateExpression(expression, frame.asyncIndex, frame.protocolId);
```

Comment:
Is this the same as leaving out the `async/await` entirely?

Reply:
Good call, think so. Let me get that in.